app/src/main/res/layout/fragment_notification.xml

```diff
@@ -15,8 +15,8 @@
             android:id="@android:id/list"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:dividerHeight="2dp"
-            android:divider="@color/accent"/>
+            android:dividerHeight="0dp"
```

Comment:
I did try that first but I found out android automatically adds a thin, light-gray divider unless you explicitly make it 0dp w/ a drawable of @null.